safe/gui/tools/wizard/test/test_wizard_util.py

```diff
@@ -5,7 +5,7 @@
 from safe.test.utilities import get_qgis_app
 QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
 
-from safe.gui.tools.wizard.wizard_utils import get_question_text
+from safe.gui.tools.wizard.utilities import get_question_text
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->